src/AWS/Lambda/RuntimeClient.hs

```diff
@@ -97,9 +97,8 @@ endpoints host =
   where baseURL' = "http://" <> host <> "/2018-06-01/runtime"
 
 forceMaybe :: String -> Maybe a -> a
-forceMaybe errorMsg maybe' = case maybe' of
-  Just a' -> a'
-  Nothing -> error errorMsg
+forceMaybe errorMsg = fromMaybe (error errorMsg)
+--forceMaybe errorMsg maybe' = fromMaybe (error errorMsg) maybe'
```

Comment:
This can be removed, right?

Reply:
yeah, it was for reference only--to show the eta reduction.